src/styles/search.css

```diff
@@ -0,0 +1,19 @@
+.search-form {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
```

Comment:
Theres a few shared properties here that could be combined into a single decalration:

```
.search-input,
.search-button {
  border-radius: 11px;
  font-size: 20px;
  padding: 0 10px;
}
```
You could then remove the `.search-button` rules as they are all covered, and remove these properties from `.search-input`